dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -722,6 +718,8 @@ public OrganizationUser addUserToOrg(@ApiParam(hidden = true) @Parameter(hidden
                 OrganizationUser.Role.valueOf(role));
             Session currentSession = sessionFactory.getCurrentSession();
             currentSession.persist(organizationUser);
+        } else if (existingRole.getStatus() == REJECTED) {
+            existingRole.setStatus(OrganizationUser.InvitationStatus.PENDING);
```

Comment:
Curious about the difference between the behavior here and in `updateUserRole` around line 756.  If the existing status is REJECTED, this code changes the status to PENDING, but the code in `updateUserRole` throws a bad request.  Why the difference?

Reply:
`addUserToOrg` handles rejected invitations by changing the status from `REJECTED` to `PENDING` to "re-add" the user to the organization and the invited user will see the invitation again under their Requests page.

For `updateUserRole`, there isn't much functionality to editing the role if the user has rejected the invitation because it can't be acted upon by the invited user (they can't view their own rejected invites), so I chose to throw a bad request